drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::Client < Types::BaseObject
+    include Types::HmisSchema::HasEnrollments
+
+    description 'HUD Client'
+    field :id, ID, null: false
+    field :personal_id, String, null: false
+    field :first_name, String, null: false
+    field :last_name, String, null: false
+    field :preferred_name, String, null: true
+    field :ssn_serial, String, null: true
+    field :dob, Types::Date, 'Date of birth as format yyyy-mm-dd', null: true
+    field :pronouns, String, null: true
+    field :date_updated, Types::DateTime, null: false
+    enrollments_field :enrollments
+    field :start_date, Types::DateTime, null: true
+    field :end_date, Types::DateTime, null: true
```

Comment:
Is there any easy way to infer big chunks of this from the existing table structure?  Something like:
```
[:first_name, :last_name].each { |f| field f, magic_get_format_from_db, null: magic_get_null_value_from_db }
```

Reply:
If nothing else, it seems like we might be able to populate large chunks of this from `app/models/concerns/hmis/structure/client.rb#hmis_configuration`

Reply:
Let's leave it as is for now.  It makes me nervous the number of places we'll need to touch when the spec changes every two years, but I don't want to slow down development with a massive abstraction right now.